main.go

```diff
@@ -1666,16 +1666,16 @@ func genCodePage(g *pageCodeGen) ([]byte, error) {
 		g.bodyPrintf("  }()\n")
 		g.used("bytes", "html/template")
 		save := g.ioWriterVar
-		g.ioWriterVar = "b"
+		g.ioWriterVar = "__pushup_b"
```

Comment:
We should probably document this (there are some other collisions possible, too), but this is definitely making it less likely to encounter in practice.

Reply:
agreed, I thought about making a full census of prohibited names but figured that could wait for a fix to free up a common one like this